packages/modules/data-widgets/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [2.2.3] Datagrid
+### Fixed
+- We fixed an issue with column hiding behaviour. Now user can't uncheck last visible column in the datagrid. (Ticket #144500)
+
```

Comment:
I don't think you have to add these manually, automation will add it to the module's changelog. You should just update the individual changelogs.

Reply:
Ok, I'll remove